src/refspec.c

```diff
@@ -160,6 +162,27 @@ void git_refspec__free(git_refspec *refspec)
 	memset(refspec, 0x0, sizeof(git_refspec));
 }
 
+int git_refspec_parse(git_refspec **refspec, const char *input, int is_fetch) {
```

Comment:
Well, this is definitly is a bit more than a "simple wrapper", regarding it has 14 lines of code. So it should be worth a test

Reply:
While you're fixing up this PR with @pks-t 's changes, please put this `{` on a newline to match our code formatting rules.  (Thanks!)

Reply:
Ah, thanks! I noticed this, but forgot to write a comment

Reply:
I'll push some tests right away.

Reply:
@pks-t About the tests, would a simple valid/not-valid parsing case be sufficient (testing object creation) and a simple comment that the logic should already be tested in the internal version of the api suffice?

Reply:
@cynex Should suffice in this case, yes.